product-properties-mappers-v1/src/commonMain/kotlin/exceptions/UnknownPropCommand.kt

```diff
@@ -0,0 +1,5 @@
+package com.crowdproj.marketplace.mappers.v1.exceptions
+
+import com.crowdproj.marketplace.common.models.PropCommand
+
+class UnknownPropCommand(command: PropCommand) : Throwable("Wrong command $command at mapping toTransport stage")
```

Comment:
Throwable has two children: Exception and Error. For ordinary custom exceptions Exception class is preferred.